app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -585,8 +585,6 @@ public static void loadLectureList(@NonNull AppRepository appRepository, int day
         MyApp.roomsMap.putAll(legacyLectureData.getRoomsMap());
         MyApp.roomList = legacyLectureData.getRoomList();
         MyApp.roomCount = legacyLectureData.getRoomCount();
-
-        loadAlarms(appRepository);
```

Comment:
Why do think we can remove the method invocation here? Please rewrite your commit message to state the reason.

Reply:
Do you have a suggestion for a message? I thought it's pretty self-explanatory that we don't have to load alarms again after just having done so while loading the lectures.

Reply:
It's clear to me now. Thank you.